bin/terraform

```diff
@@ -1,7 +1,22 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+args=( "$@" )
+
+args+=("-var-file=/config/config.tfvars")
+
+if [[ "$1" == "init" ]] ; then
+    args+=(
+        "-backend-config=bucket=${ACTIVE_SITE}-terraform"
+        "-backend-config=key=${ACTIVE_SITE}.tfstate"
+        "-backend-config=dynamodb_table=${ACTIVE_SITE}_terraform_state_locks"
+    )
+fi
+
 # Run Terraform commands in the Hashicorp docker container, passing our
 # environment variables
-DOCKER_USER="$(id -u):$(id -g)" \
-    docker-compose run --rm terraform "$@"
+DOCKER_USER="$(id -u):$(id -g)" docker-compose run --rm terraform \
+    "${args[@]}"
```

Comment:
I find previous indentation more readable.

Reply:
Well... shellcheck didn't like it so I found out that everything fits in one line...